VimeoNetworking/VimeoNetworking/Models/VIMVideo.m

```diff
@@ -437,7 +437,16 @@ - (BOOL)isRatedMature
 
 - (BOOL)isDRMProtected
 {
-    return self.playRepresentation.drmFiles.fairPlayFile != nil;
+    VIMInteraction *buyInteraction = [self interactionWithName:VIMInteractionNameBuy];
```

Comment:
This logic is in two classes, is there a way to consolidate it so it only appears once?